src/lib/holocene/calendar.svelte

```diff
@@ -66,11 +66,11 @@
   }
 
   .cell {
-    @apply m-1 inline-flex h-[24px] w-[24px] items-center justify-center rounded p-1 text-sm;
+    @apply m-1 inline-flex h-[24px] w-[24px] items-center justify-center rounded-2xl p-1 text-sm text-primary;
   }
 
   .selected {
-    @apply bg-gradient-to-br from-blue-100  to-purple-100 text-primary;
+    @apply surface-interactive rounded-2xl  p-2 text-offWhite dark:bg-indigo-300 dark:text-black;
```

Comment:
Looks like the hover and selected style is the same in light mode ⬇️ 

<img width="128" alt="Screenshot 2024-04-05 at 11 40 45 AM" src="https://github.com/temporalio/ui/assets/15069288/35214b14-52f6-42fb-9081-8e73601535f4">

Might we want to make them slightly different similar to how they are in dark mode?

<img width="111" alt="Screenshot 2024-04-05 at 11 40 13 AM" src="https://github.com/temporalio/ui/assets/15069288/8e5402b5-cb3b-4736-914f-bf58a4300b7f">

Reply:
Yes, this is from the designs. I will ask Ash if they want to change this.